lib/msal-common/src/utils/Constants.ts

```diff
@@ -32,6 +32,15 @@ export const Constants = {
     URL_FORM_CONTENT_TYPE: "application/x-www-form-urlencoded"
 };
 
+/**
+ * Denotes which way MSAL is being used per request
+ */
+export enum MsalMethod {
+    SILENT = "silent",
+    POPUP = "popup",
+    REDIRECT = "redirect"
```

Comment:
And then have this is browser and be part of the computed state (which we know will be a more complicated base64 encoded string)

Reply:
so do we want to keep this or simplify?

Reply:
@sameerag i think the idea is to keep this, but move it to browser and make the code in msal common (on jason's other comment) be more generic